### PR TITLE
SFL header changes

### PR DESCRIPTION
**Pull request by OliverJAsh**
image

Leaving the styles inside the drop down as they are right now. Will improve after test.

### CODE REVIEW

**Comment by NathanielBennett** [comment]
:+1:

File: static/src/javascripts/projects/common/modules/experiments/tests/save-for-later.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ define([
 
                     mediator.on('modules:profilenav:loaded', function () {
                         var popup = qwery('.popup--profile')[0];
-                        bonzo(popup).append(bonzo.create(
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/experiments/tests/save-for-later.js]: fastdom

File: static/src/javascripts/projects/common/modules/save-for-later.js [hunk 2]
```diff
@@ -320,7 +320,12 @@ define([
         var self = this,
             saveForLaterProfileLink = $(self.classes.profileDropdownLink);
 
-        saveForLaterProfileLink.html('Saved (' + self.userData.articles.length + ')');
+        var count = self.userData.articles.length;
+        saveForLaterProfileLink.html('Saved (' + count + ')');
+
+        var profile = $('.brand-bar__item--profile');
+        profile.addClass('brand-bar__item--profile--show-saved');
```
Comment (sndrs) [inline static/src/javascripts/projects/common/modules/save-for-later.js]: fastdom

**Comment by sndrs** [comment]
:+1: other than comments

**Pull request merged**